packages/gas-fee-controller/src/gas-util.test.ts

```diff
@@ -65,27 +72,51 @@ const mockEIP1559ApiResponses: GasFeeEstimates[] = [
   },
 ];
 
+const INFURA_AUTH_TOKEN_MOCK = 'test';
+const INFURA_GAS_API_URL_MOCK = 'https://gas.api.infura.io';
+
 describe('gas utils', () => {
+  let handleFetchMock: jest.Mock;
+
+  beforeEach(() => {
+    handleFetchMock = handleFetch as jest.Mock;
+    handleFetchMock.mockReset();
+  });
+
```

Comment:
Since `handleFetch` is accessible from any test, what are your thoughts on moving this variable to the top, and using `jest.mocked` instead of using a type assertion? Conventionally in other files we've put it after the `jest.mock`, so you could say:

``` typescript
jest.mock('@metamask/controller-utils', () => {
  return {
    ...jest.requireActual('@metamask/controller-utils'),
    handleFetch: jest.fn(),
  };
});

const handleFetchMock = jest.mocked(handleFetch);
```

Also, the need to reset this mock function reminds that we've configured Jest to not reset all mock functions, so it may be better to do that here instead:

``` suggestion
  beforeEach(() => {
    jest.resetAllMocks();
  })

Reply:
That make sense Elliot, fixed in https://github.com/MetaMask/core/pull/4068/commits/7301ac99a99f58891ae0b5ee7a1b27fa7f257bd0